packet/packet_composed/src/composed.cpp

```diff
@@ -0,0 +1,109 @@
+#include <chrono>
+#include <functional>
+#include <string>
+#include "packet_composed/sensorspacket.hpp"
+
+
+Composed::Composed() :
+    rclcpp::Node("composed"),
+    _publisher(),
+    _timer(),  
+    _count(0)  
+{
+    //composed_publisherをコンストラクタで初期化した。
+    using namespace std::chrono_literals;
+    _publisher = this->create_publisher<std_msgs::msg::String>("/sensors_composed", 10);
+    composed_publisher = this->create_publisher<packet_interfaces::msg::Composed>("composed_topic", 10);
+
+    depth_subscription = this->create_subscription<packet_interfaces::msg::Depth>(
+        "depth_topic", 10, std::bind(&Composed::depth_topic_callback, this, std::placeholders::_1));
+
+    imu_subscription = this->create_subscription<sensor_msgs::msg::Imu>(
+        "imu_topic", 10, std::bind(&Composed::imu_topic_callback, this, std::placeholders::_1));
+
+    flex1_subscription = this->create_subscription<packet_interfaces::msg::Flex>(
+        "flex1_topic", 10, std::bind(&Composed::flex1_topic_callback, this, std::placeholders::_1));
+
+    flex2_subscription = this->create_subscription<packet_interfaces::msg::Flex>(
+        "flex2_topic", 10, std::bind(&Composed::flex2_topic_callback, this, std::placeholders::_1));
+
+    current_subscription = this->create_subscription<packet_interfaces::msg::Current>(
+        "current_topic", 10, std::bind(&Composed::current_topic_callback, this, std::placeholders::_1));
+
+    voltage_subscription = this->create_subscription<packet_interfaces::msg::Voltage>(
+        "voltage_topic", 10, std::bind(&Composed::voltage_topic_callback, this, std::placeholders::_1));
+   
+    auto loop = std::bind(&Composed::_loop, this);
+    _timer = this->create_wall_timer(500ms, loop);
+}
+
+void Composed::_loop() {
+    std::stringstream ss;
+    ss << "Hello, world! " << _count;
+    std_msgs::msg::String msg{};
+    msg.data = ss.str();
+    RCLCPP_INFO(this->get_logger(), "say %s", ss.str().c_str());
+    _publisher->publish(msg);
+    _count++;
+}
+
+void Composed::depth_topic_callback(const packet_interfaces::msg::Depth& msg)
+{
+    RCLCPP_INFO(this->get_logger(), "Received Depth message");
```

Comment:
ログのレベルはデバッグでお願いします(他のcallback部分も同様)

```suggestion
    RCLCPP_DEBUG(this->get_logger(), "Received Depth message");
```